### DISCUSSION TITLE
혹시.. 컴퓨터에서 네이버 플레이스 지도화면이 안뜨는 이유가뭘까요? 도와주세요ㅠㅠ

### DISCUSSION THREAD

**Discussion by MingyuTheAnalyst** [changdoc/naver-vaccine-macro#136] [Q&A]
크롬, 엣지, 웨일 모두 다 네이버플레이스 지도가 들어가지지가 않네요..

이유를 찾을수가 없어요..

화면은 이렇게만 뜨고 더이상 들어가지질 않아서 진행을 못하고있어요 ㅠㅠ도와주세요

![Cap 2021-07-29 10-34-14-156](https://user-images.githubusercontent.com/88122148/127420983-392a9012-e430-4dcf-89ad-7c841869a531.png)

**Comment by changdoc** [comment]
개발자도구를 여셔서 console창에 무슨 에러가 난건지 확인이 필요해보이는데요.
네이버측에 문의를 하는게 더 빠르실 것 같습니다..
